src/cloud/project/project-webint-snap.md

```diff
@@ -163,9 +163,10 @@ To create a database dump:
 
  {:.bs-callout-info}
 
--  We recommend putting the application in maintenance mode before doing a database dump in Production environments.
+-  Before creating dump the command will put the application in maintenance mode. Also, command stops all crons and kill all running consumers and crons.   
 -  This command creates an archive file for each database backup with the file name pattern `dump-<label><timestamp>.sql.gz` where  _label_ is replaced with the database name.  The archive files are saved to your remote project directory, and the path to each file is listed in the command output.
 -  If an error occurs during the dump, the command deletes any dump files to conserve disk space. Review the logs for details (`var/log/cloud.log`).
```

Comment:
If the dump fails, do you have to do something about switching back to production mode and restarting cron processes?

Reply:
Yes, in case of fail command will switch back application to production and start crons.